src/main/java/io/pravega/perf/PravegaPerfTest.java

```diff
@@ -277,6 +281,12 @@ public static Test createTest(long startTime, CommandLine commandline, Options o
                 recreate = producerCount > 0 && consumerCount > 0;
             }
 
+            if (commandline.hasOption("fork")) {
```

Comment:
You can simplify this to
`enableConnectionPooling = Boolean.parseBoolean(commandline.getOptionValue("enableConnectionPooling", "true"));
`

Reply:
yes, its done.. I covered it for all other parameters too.